nevergrad/optimization/optimizerlib.py

```diff
@@ -1143,10 +1143,15 @@ def learn_on_k_best(archive: utils.Archive[utils.MultiValue], k: int) -> tp.Arra
     model = LinearRegression()
     model.fit(X2, y)
 
-    optimizer = Powell(parametrization=dimension, budget=45*dimension+30)
     try:
-        minimum = optimizer.minimize(
-            lambda x: float(model.predict(polynomial_features.fit_transform(x[None, :])))).value
+        for cls in Powell, DE:  # Powell excellent here, DE as a backup for thread safety.
```

Comment:
Compared to PR 923, here I add this backup: if Powell crashes, we use DE, which is also able to deal with ill-conditioning and is super stable.

Not sure if I should do something deeper than that. I don't like this solution.